providers/shared/components/component.ftl

```diff
@@ -594,6 +594,27 @@
                     "Default" : "waf"
                 }
             ]
+        },
+        {
+            "Names" : "RateLimits",
+            "Children" : [
+                {
```

Comment:
Why the IP section? Doesn't the IPAddressGroups imply that? 
Are you thinking of implementing other ways to rate limit?

Reply:
Good point, ive removed the unnecessary level in 8a3410f